build/packages/deb/postrm

```diff
@@ -22,8 +22,23 @@ esac
 
 if [ "$1" = "purge" ]; then
     userdel pmm-agent || true
-    if [ -f /usr/local/percona/pmm2/config/pmm-agent.yaml ]; then
-        rm -f /usr/local/percona/pmm2/config/pmm-agent.yaml
+    if [ -f /usr/local/percona ]; then
+        echo -n "Deleting percona directory..."
+        rm -f /usr/local/percona
+        echo "OK"
```

Comment:
This type of message is not informative I'm afraid. `Done deleting ... directory` is much better.